onadata/apps/api/management/commands/delete_users.py

```diff
@@ -0,0 +1,126 @@
+"""
+Delete users management command.
+"""
+import sys
+from django.contrib.auth.models import User
+from onadata.apps.logger.models import XForm
+from onadata.apps.logger.models import Instance
+from onadata.apps.logger.models import Project
+from django.core.management.base import BaseCommand, CommandError
+from django.utils import timezone
+
+
+def get_user_object_stats(
+        username, email, user_input=None):  # pylint: disable=R0201
+    """
+    Get User information.
+    """
+    # Get the number of projects for this user
+    user_projects = Project.objects.filter(
+        created_by__username=username).count()
+    # Get the number of forms
+    user_forms = XForm.objects.filter(
+        user__username=username).count()
+    # Get the number of submissions
+    user_sumbissions = Instance.objects.filter(
+        user__username=username).count()
+    result = {
+        'projects': user_projects,
+        'forms': user_forms,
+        'submissions': user_sumbissions
+    }
+
+    user_response = input(
+        "User account '{}' has {} projects, "
+        "{} forms and {} submissions. "
+        "Do you wish to continue "
+        "deleting this account?".format(
+            username,
+            result['projects'],
+            result['forms'],
+            result['submissions']
```

Comment:
I think using `user_projects, user_forms, user_submissions` instead of setting `result` would be nice here ? `result` doesn't seem to be used anywhere else...

Reply:
changed that